index.html

```diff
@@ -1261,66 +1261,68 @@ <h2>Characters</h2>
 <div id="char_checklist"></div>
 </details-->
 
+<section id="char_def" class="subtopic">
```

Comment:
You seem to have lost the 'See also' panel that is in the original chapter text.

Reply:
We have a link to that section in the body of the text. I could make see-also of that link and the one on choosing encodings in appropriate places instead of in-lining the reference.